src/Stratis.Bitcoin.Features.Consensus/ConsensusLoop.cs

```diff
@@ -73,7 +74,7 @@ public class BlockValidationContext
     /// <remarks>
     /// <para>
     /// Blocks are coming from <see cref="ILookaheadBlockPuller"/> or Miner/Staker and get validated by
-    /// either the <see cref="PowConsensusValidator"/> for PoW or the <see cref="PosConsensusValidator"/> for PoS.
+    /// either the <see cref="PowCoinviewRule"/> for PoW or the <see cref="PosCoinviewRule"/> for PoS.
```

Comment:
"And get validated by the IConsensusRules engine"

Reply:
done, I also changed the line there that points to the coinview rules to....

/// See either the <see cref="FullNodeBuilderConsensusExtension.PowConsensusRulesRegistration"/> for PoW or the <see cref="FullNodeBuilderConsensusExtension.PowConsensusRulesRegistration"/> for PoS.